mon-pix/app/components/user-certifications-detail-header.js

```diff
@@ -23,7 +24,7 @@ export default class UserCertificationsDetailHeader extends Component {
   @action
   downloadAttestation() {
     const certifId = this.args.certification.id;
-    const url = `/api/attestation/${certifId}`;
+    const url = `/api/attestation/${certifId}?isFrenchDomainExtension=${this.url.isFrenchDomainExtension}`;
```

Comment:
```suggestion
    const url = `/api/attestation/${certifId}?shouldDisplayProfessionalizingWarning=${this.args.shouldDisplayProfessionalizingWarning}`;
```

Il y'a un getter dans le modèle de certification qui retourne true ou false selon le domaine de l'url et la date. ça serait peut être interessant de s'en servir ici

Reply:
Ce n'est pas au front de prendre cette decision